app/controllers/collections_controller.rb

```diff
@@ -173,6 +170,30 @@ def destroy
 
   private
 
+  def parsed_collection_filters
+    # find fandom id from name given in filter, otherwise send params through
+    parsed_filters = collection_filter_params.inject({}) do |new_hash, (k, v)|
+      if k == 'fandom'
+        fandom = Fandom.find_by(name: v)
+        new_hash[:fandom_ids] = [fandom&.id].compact
+      else
+        new_hash[k] = v
+      end
+      
+      new_hash
+    end 
+
+    parsed_filters.symbolize_keys
+  end
+
+  def collection_filter_params
+    white_list = %w(title fandom challenge_type moderated closed sort_column sort_direction)
```

Comment:
We're trying to avoid whitelist/blacklist terminology going forward... can we use `safelist` instead? That's what we went with in #3825